addons/binding/org.openhab.binding.pulseaudio/src/main/java/org/openhab/binding/pulseaudio/internal/cli/Parser.java

```diff
@@ -16,384 +16,367 @@
 import java.util.regex.Pattern;
 
 import org.openhab.binding.pulseaudio.internal.PulseaudioClient;
-import org.openhab.binding.pulseaudio.internal.items.*;
+import org.openhab.binding.pulseaudio.internal.items.AbstractAudioDeviceConfig;
+import org.openhab.binding.pulseaudio.internal.items.Module;
+import org.openhab.binding.pulseaudio.internal.items.Sink;
+import org.openhab.binding.pulseaudio.internal.items.SinkInput;
+import org.openhab.binding.pulseaudio.internal.items.Source;
+import org.openhab.binding.pulseaudio.internal.items.SourceOutput;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 /**
  * Parsers for the pulseaudio return strings
- * 
+ *
  * @author Tobias Bräutigam
  * @since 1.2.0
  */
 public class Parser {
-	private static final Logger logger = LoggerFactory
-			.getLogger(Parser.class);
-	
-	private static final Pattern pattern = Pattern
-			.compile("^\\s+([a-z\\s._]+)[:=]\\s*<?\"?([^>\"]+)\"?>?$");
-	private static final Pattern volumePattern = Pattern
-			.compile("^0:\\s*([0-9]+)%\\s*1:\\s*([0-9]+)%.*$");
-	private static final Pattern fallBackPattern = Pattern
-			.compile("^([0-9]+)([a-z\\s._]+)[:=]\\s*<?\"?([^>\"]+)\"?>?$");
-	private static final Pattern numberValuePattern = Pattern
-			.compile("^([0-9]+).*$");
+    private static final Logger logger = LoggerFactory.getLogger(Parser.class);
```

Comment:
How can I use the logger from within the static methods, when its not declared as static?